pom.xml

```diff
@@ -21,7 +21,7 @@
         <module>de.tubs.variantsync.core</module>
         <module>de.tubs.variantsync.core.feature</module>
         <module>de.tubs.variantsync.update</module>
-        <module>de.variantsync.core</module>
+        <!--<module>de.variantsync.core</module>-->
```

Comment:
What does this mean? Don't you build the library anymore?

Reply:
Read the section "linking plugin and library"  in the wiki: https://github.com/tcerny/VariantSync/wiki/Maven#linking-plugin-and-library